src/commands/registerCommands.ts

```diff
@@ -93,7 +92,6 @@ export function registerCommands(): void {
     registerCommandWithTreeNodeUnwrapping('azureFunctions.pickProcess', pickFuncProcess);
     registerCommandWithTreeNodeUnwrapping('azureFunctions.redeploy', redeployDeployment);
     registerCommandWithTreeNodeUnwrapping('azureFunctions.restartFunctionApp', restartFunctionApp);
-    registerCommandWithTreeNodeUnwrapping('azureFunctions.setAzureWebJobsStorage', setAzureWebJobsStorage);
```

Comment:
Don't we still need this for the command palette to work?

Reply:
You are absolutely correct. Forgot to revert that one. 🤦